src-test/org/etools/j1939_84/bus/j1939/J1939TPTest.java

```diff
@@ -25,6 +26,18 @@
 
 public class J1939TPTest {
 
+    static private void assertPacketsEquals(Collection<Packet> expected, Collection<Packet> actual) {
+        Assert.assertEquals(expected.toString(), actual.toString());
+    }
+
+    static private void assertPacketsEquals(Packet expected, Packet actual) {
+        Assert.assertEquals(expected.toString(), actual.toString());
```

Comment:
it seems like it might be better to write equals() on packet, but this works.

Reply:
equals() needs a context.  Does the timestamp count?  This is the exact comparison we need for these tests.

Reply:
Oh, right. The time of each packet is different. Nevermind